src/layout/flex-grid/src/index.tsx

```diff
@@ -191,3 +190,40 @@ export const Col: React.FC<ColProps> = ({
     </div>
   )
 }
+
+interface FromJsonProps extends React.HTMLAttributes<any> {
+  json: (RowProps & {
+    layout: (ColProps & {
+      key: number
+    })[]
+  })[]
+  childrenArray: React.ReactNode[]
+}
+
+export const GridFromJson: React.FC<FromJsonProps> = ({
+  json,
+  childrenArray
+}) => {
+  // In the future, string key could be support for contentful block ID
+  const getChildFromKey = (key: number): React.ReactNode => {
+    return childrenArray[key]
+  }
+
+  return (
+    <div>
```

Comment:
Probably better to use a Fragment here, unless you need the div for layout for some reason?